Recast/HomeViewController.swift

```diff
@@ -0,0 +1,298 @@
+        return HomeSectionType.allCases.count
+    }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let headerView = HomeTableViewHeaderView()
+        if let sectionType = HomeSectionType.type(for: .section, value: section) {
+            headerView.configure(for: sectionType)
+        }
+        return headerView
+    }
+
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 60
+    }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 108
+    }
+}
+
+// MARK: - TV Data Source
+extension HomeViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let viewTag = HomeSectionType.type(for: .section, value: indexPath.section)!.tag
+        switch indexPath.section {
+        case 0:
+            // swiftlint:disable:next force_cast
+            let cell = tableView.dequeueReusableCell(withIdentifier: continueTvReuse) as! ContinueListeningTableViewCell
+            cell.collectionView.delegate = self
+            cell.collectionView.dataSource = self
+            cell.collectionView.tag = viewTag
+            cell.collectionView.reloadData()
+            return cell
+        case 1, 2:
+            // swiftlint:disable:next force_cast
+            let cell = tableView.dequeueReusableCell(withIdentifier: gridTvReuse) as! PodcastGridTableViewCell
+            cell.collectionView.delegate = self
+            cell.collectionView.dataSource = self
+            cell.collectionView.tag = viewTag
+            cell.collectionView.reloadData()
+            return cell
+        default:
+            return UITableViewCell(frame: .zero)
+        }
+    }
+
+}
+
+// MARK: - CV Delegate
+extension HomeViewController: UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        let type = HomeSectionType.type(for: .tag, value: collectionView.tag)!
+        switch type {
+        case .continueListening:
+            break
+        case .yourFavorites:
+            break
+        case .browseTopics:
+            break
+        }
+    }
+}
+
+// MARK: - CV Data Source
+extension HomeViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        let type = HomeSectionType.type(for: .tag, value: collectionView.tag)!
+        switch type {
+        case .continueListening:
+            return continuePodcasts.count
+        case .yourFavorites:
+            return favoritePodcasts.count
+        case .browseTopics:
+            return browsePodcasts.count
+        }
+    }
+
+    func collectionView(
```

Comment:
woah what happened here? does this need to be broken up like this for swiftlint?